storage/__init__.py

```diff
@@ -1242,6 +1242,10 @@ def useExisting(lst):
             return (foundCreateFormat and not foundCreateDevice)
 
         log.warning("Storage.writeKS not completely implemented")
+
+        if self.zeroMbr:
+            f.write('# Clear the Master Boot Record\nzerombr\n')
```

Comment:
Please split this to two lines. I think it will be better to find on separate line.